airflow/providers/oracle/hooks/oracle.py

```diff
@@ -251,8 +251,7 @@ def insert_rows(
         table: str,
         rows: list[tuple],
         target_fields=None,
-        commit_every: int = 1000,
-        replace: bool | None = False,
+        commit_every: int = 1000
```

Comment:
You can't remove `replace` argument, because it is comes from the `DbApiHook. insert_rows`

https://github.com/apache/airflow/blob/08f4b923ab6fe63aad72e3a9da9507ed5b9c6932/airflow/providers/common/sql/hooks/sql.py#L555-L565 

If you would like to inform user that it doesn't  take any affect for OracleHook you could inform by:
- Mentioned it into the docstring
- Warn users by appropriate warning message: `warning.warn("Foo bar do not use that", category=UserWarning, stacklevel=2)`

Reply:
I've reintroduced the replace parameter and implemented both suggestions to inform the user. Is the current version fine?